### PR TITLE
Support for io.js

### PR DESCRIPTION
**Pull request by jkrems**
Verifying that https://github.com/groupon-testium/webdriver-http-sync/pull/26 does not break testium.

### CODE REVIEW

File: test/integration/unicode.test.coffee [hunk 1]
```diff
@@ -12,5 +12,5 @@ describe 'unicode support', ->
     element = @browser.getElement '#blank-input'
     element.type multibyteText
     result = element.get('value')
-    assert.equal result, multibyteText
+    assert.equal multibyteText, result
```
Comment (jkrems) [inline test/integration/unicode.test.coffee]: The argument order here was wrong which lead to slightly confusing error messages.

**Comment by jkrems** [comment]
The test failures seem not to be connected to the webdriver and node 0.10 (the version we officially support currently) is green.

**Comment by EndangeredMassa** [comment]
What test failures?

**Comment by jkrems** [comment]
The ones I hid away by disabling `img-diff` tests on io.js and node 0.12 (`png` still doesn't compile).

**Comment by EndangeredMassa** [comment]
I think we should deprecate img-diff. The current implementation it too slow to really use anyway.

**Comment by EndangeredMassa** [comment]
Let me know if we should merge this or if we need to wait on more changes.

**Comment by jkrems** [comment]
Thanks for the ping! Kind of lost track. Removed the git reference from the dependencies. I think this is ready to review/merge now.

**Comment by jkrems** [comment]
Not sure why the build isn't stable. :(

**Comment by EndangeredMassa** [comment]
closing in favor of: https://github.com/groupon-testium/testium/pull/157

**Pull request closed without merging**